static/src/javascripts/projects/common/modules/experiments/tests/commercial-consent-global-banner.js

```diff
@@ -1,20 +1,20 @@
 // @flow
 import { getCookie } from 'lib/cookies';
 
-export const commercialConsentGlobalNoScroll: ABTest = {
-    id: 'CommercialConsentGlobalNoScroll',
-    start: '2019-04-24',
-    expiry: '2019-05-24',
+export const commercialConsentGlobalBanner: ABTest = {
+    id: 'CommercialConsentGlobalBanner',
+    start: '2019-05-01',
+    expiry: '2019-05-31',
     author: 'George Haberis',
     description:
-        'Test the consent banner on non-EEA users with & without the ability to scroll underlying content through the banner on mobile',
-    audience: 0.02,
+        'Test whether changes to Consent Banner increases proportion of non-EU users who interact with it',
+    audience: 0.03,
     audienceOffset: 0.1,
     successMeasure: 'Users outside of the EU interact with the consent banner',
     audienceCriteria: 'all users',
     dataLinkNames: '',
     idealOutcome:
-        'More non-EEA users interact with the consent banner when we prevent the ability to scroll underlying content through the banner on mobile',
+        'More non-EEA users interact with the consent banner when we alter ir',
```

Comment:
Typo here? `ir` -> `it`. Might be worth wording this as something like `fewer non-EEA users ignore the altered banner, which reduces the number of pageviews with unset conset`